ts_src/index.ts

```diff
@@ -73,7 +73,10 @@ export function mnemonicToSeedSync(
   mnemonic: string,
```

Comment:
To make the type accurate:
```suggestion
  mnemonic: Buffer | string,
```

Reply:
done here: https://github.com/MetaMask/bip39/pull/1/commits/f5dcd825314b3e9b32e28bf0eb51ae8a9d27e240

Reply:
scratch that. Done here: https://github.com/MetaMask/bip39/pull/1/commits/a01d40d19791de9cee60a80d87d35b61feb6dc97